lib/pf/radius.pm

```diff
@@ -108,11 +112,10 @@ sub authorize {
     # let's check if an old port sec entry needs to be removed in another switch
     $this->_handleStaticPortSecurityMovement($switch, $mac);
```

Comment:
Me is blind, me no see it.

I know the authorize sub is timed but I don't see this one being timed explicitly.

But I may be blind.

I'll reboot my computer to see if it helps